src/fitnesse/testsystems/slim/SlimClientBuilder.java

```diff
@@ -92,9 +94,10 @@ protected String defaultTestRunner() {
     	arguments = ArrayUtils.add(arguments, "-ssl");
     	arguments = ArrayUtils.add(arguments, useSSL);
     }    	
-    String slimFlags = getSlimFlags();
-    if (!"".equals(slimFlags))
-    	arguments = ArrayUtils.add(arguments, slimFlags);
+    String slimFlags[] = getSlimFlags();
+    if (slimFlags != null)
+    	for (String flag : slimFlags)
+    		arguments = ArrayUtils.add(arguments, flag);
```

Comment:
The method ClientBuilder.parseCommandLine() could be used to parse the flags. That will take into account flags containing quotes.